mk/main.mk

```diff
@@ -391,18 +391,10 @@ ifeq ($$(CFG_WINDOWSY_$(3)),1)
 # there's no rpath. Target libraries go under $CFG_LIBDIR_RELATIVE (usually 'lib').
 HLIB_RELATIVE$(1)_H_$(3) = bin
 TROOT$(1)_T_$(2)_H_$(3) = $$(HROOT$(1)_H_$(3))/$$(CFG_LIBDIR_RELATIVE)/rustlib/$(2)
-# Remove the next 3 lines after a snapshot
-ifeq ($(1),0)
-RUSTFLAGS_STAGE0 += -L $$(TROOT$(1)_T_$(2)_H_$(3))/lib
-endif
 
 else
 
-ifeq ($(1),0)
-HLIB_RELATIVE$(1)_H_$(3) = lib
-else
```

Comment:
Hm I was [unable to reproduce the errors](https://github.com/rust-lang/rust/issues/32261#issuecomment-196960498), so I want to dig a little more into what's going on here.